scripts/apps/authoring/authoring/services/RenditionsService.ts

```diff
@@ -91,12 +91,6 @@ export function RenditionsService(metadata, $q, api, superdesk, _) {
                 .then((metaData) => {
                     // apply the metadata changes
                     return _.extend(item, metaData);
-                }).catch((response) => {
-                    // if new crops not generated continue with default one
-                    // see https://github.com/superdesk/superdesk-client-core/pull/3117#discussion_r328440897
```

Comment:
I hope you have checked that you aren't breaking the behaviour from another PR

Reply:
I'm waiting for Gyan to review the PR. I did check it but I want him to confirm with him